chess-AI/train.py

```diff
@@ -6,7 +6,7 @@
 import torch
```

Comment:
https://pastebin.com/bVTWKNkC

Reply:
Again , not sure what pylint your using but I ran pylint on it and got a 8.57/10. The only issues I got was code that we already had on master such as too many branhe and too many local variables.

Reply:
I think it might have to do with your `~/.pylintrc` file not including enough stuff. Can you past the contents of this link (https://pastebin.com/qmXVMj3P) into that file and tell me if you are able to see more verbose errors?

Reply:
Lol okay I can try that when I have the time. In the meantime if anyone wants to still review the functionality of the code , the pylint issues just seem to deal with small things like spacing.